README.md

```diff
@@ -86,8 +86,7 @@ $ bash util/install_macos.sh
 You can also use [homebrew](https://brew.sh/) as an alternative:
 
 ```bash
-brew tap homebrew/cask-fonts
-brew install font-monaspace
+$ brew install --cask font-monaspace
```

Comment:
I believe specifying `cask` is deprecated/obsolete?

```suggestion
$ brew install font-monaspace
```

Reply:
This should be correct -- I grabbed the command directly from https://formulae.brew.sh/cask/font-monaspace.

Maybe you're thinking of `brew cask` which was removed?

Reply:
I believe I saw discussions that `--cask` can and should be ommitted, in the Homebrew repo. But 🤷‍♀️ 

Anyhow, it does detect automagically that it's a cask and no formula ;-)

<img width="1469" alt="Screenshot 2024-06-07 at 19 07 37" src="https://github.com/githubnext/monaspace/assets/16012374/8ed55e00-62f0-4344-ba11-127cb2be798c">


Because casks and formulae are handled the same and both are searched if nothing is specified:
https://docs.brew.sh/Manpage#install-options-formulacask-

The option is only there to specify what you want in case there is an equal named cask **and** formula.

<img width="766" alt="Screenshot 2024-06-07 at 19 18 07" src="https://github.com/githubnext/monaspace/assets/16012374/b1fe969a-0045-4a68-8f75-4dca527acc30">

---

Anyhow, this is nitpicking, the PR is also good to pull as it is, its more a matter of taste.

👍